packetery/src/Packetery/Plugin.php

```diff
@@ -158,7 +157,7 @@ public function __construct(
 	 * Method to register hooks
 	 */
 	public function run(): void {
-		$this->load_textdomains();
+		add_action( 'init', array( $this, 'loadTranslation' ), -1000 );
```

Comment:
co znamená poslední parametr a proč má hodnotu -1000 ?

Reply:
to je pořadí ve frontě. Přes add_action přidáváš do fronty callback, který se spustí, až se wordpress rozhodne. Třetí argument minimalizuje riziko, že se něco zaregistruje před načítáním překladů.

Reply:
Nechal bych defaultní parametr 10 (tedy neuvádět prioritu), pokud nevíme přesně, proč potřebujeme prioritu -1000

Reply:
Po dohodě nastavme prioritu na hodnotu 1